cmd/run.go

```diff
@@ -107,6 +109,30 @@ a commandline interface for interacting with it.`,
 			Left:  func() string { return "    init" },
 		}
 
+		// Load plugins
+		cliConf, err := getConfig(cmd.Flags())
+		if err != nil {
+			return err
+		}
+
+		plugins := []plugin.JavaScriptPlugin{}
+		pluginNames := []string{}
+		for _, pluginPath := range cliConf.Plugins {
+			jsPlugin, err := lib.LoadJavaScriptPlugin(pluginPath)
+			if err != nil {
+				return err
+			}
+
+			// Run plugin setup and add it to the module tree
+			if err = jsPlugin.Setup(); err != nil {
```

Comment:
We might want to consider running this in a separate goroutine with some timeout, to avoid a misbehaving plugin from blocking further execution, and maybe load them concurrently before the `Runner` is created.

Reply:
> We might want to consider running this in a separate goroutine with some timeout

Added to the list :-)